sdk/core/Azure.Core/samples/Serialization.md

```diff
@@ -1,34 +1,48 @@
-# Azure.Core public serialization samples
+# Azure.Core Public Serialization Guide
```

Comment:
```suggestion
# Azure.Core public serialization
```

Our docs follow the [Microsoft style guide](https://learn.microsoft.com/en-us/style-guide/capitalization#sentence-style-capitalization-in-titles-and-headings) which calls for sentence-style capitalization in headers.

Reply:
@nisha-bhatia This comment reminded me of something. You can run [Acrolinx in VS Code](https://aka.ms/azsdk/acrolinx-vscode) to check this doc against many of the style guide rules. It's licensed in our repo.